### ISSUE TITLE
Install CATS Firefox Toolbar 

### ISSUE THREAD

**Issue by amaisonneuve** [opencats/OpenCATS#29]
This extension isn't available in OpenCATS, do we have it anywhere or should this page be removed?

**Comment by RussH** [comment]
It certainly used to be available.. I've not tried it for some years.

**Comment by amaisonneuve** [comment]
The pages points to CATSONE.com

xpi["CATS ToolBar"] = "http://www.catsone.com/extensions/firefox/catstoolbar.xpi";

and that directory is not available and there is no 'extensions' directory in the REPO.

**Comment by RussH** [comment]
I have have saved it someplace - let me check

**Comment by RussH** [comment]
Okay, the historic toolbars are here - but they are for the hosted version of cats and would need some rework to point at a local opencats installation;

http://ftp.nchu.edu.tw/Unix/Mozilla/addons/180645/ 

Interestingly, also found the first 'hosted cats' outlook connector.

**Comment by amaisonneuve** [comment]
Ok perfect, i will make the required changes to my fork and will sent a pull request.

Oh is you have the connector i would love to take a look at it also.

**Comment by RussH** [comment]
I'll check some ancient desktop PC's I have if they still boot. I may have the original 'local' toolbar on there.

**Comment by puneetrinity** [comment]
Hello Russh,

_Okay, the historic toolbars are here - but they are for the hosted version of cats and would need some rework to point at a local opencats installation;

http://ftp.nchu.edu.tw/Unix/Mozilla/addons/180645/

Interestingly, also found the first 'hosted cats' outlook connector.
_

The above link is not working..
Please Help

**Comment by RussH** [comment]
Here's a mirror..

http://kyoto-mz-dl.sinet.ad.jp/pub/mozilla.org/addons/180645/

**Comment by puneetrinity** [comment]
Anyone Interested in creating a webcrawler which can be synced with an ats to source profiles?

**Comment by puneetrinity** [comment]
@RussH Thanks for the ATS-Opencats its amazing and works perfect..

**Comment by ManuelNL** [comment]
I think crawling the Web will result in a lot of rubbish. Crawling specific sites and easly copying results into your ATS  could be a nice feature! (The Toolbar? :-) )

**Comment by RussH** [comment]
Yes, if you keep an eye on commits, there should be a linkedin / monster
toolbar in the near future



----
Russ

**Comment by puneetrinity** [comment]
Hello There,
I have developed a tool (mozilla extension) which can scrap data from linkedin(Find email). Can anyone suggest how to map the Github and facebook profile of the person (without crawling the web) utilizing his email and contact details.
Google API can be used but accuracy is an issue as many a times the first result is not the correct match for the person. 
Please suggest any other way.

**Comment by wjcheers** [comment]
@RussH 
The above link is not working..
Could you share it again?

**Comment by RussH** [comment]
The old Firefox toolbar integration is no longer present in current OpenCATS, so this issue is no longer applicable. Closing.
